stripe/_util.py

```diff
@@ -232,6 +234,10 @@ def convert_to_stripe_object(
     # the raw API response information
     stripe_response = None
 
+    # Imports here at runtime to avoid circular dependencies
+    from stripe._stripe_response import StripeResponse
```

Comment:
Is this a new circular dependency? Should we move deprecation into it's own file?

Reply:
`util` and `stripe_object` are already circular but no errors happened because existing usages of util inside `stripe_object` were not top-level.

IMO `deprecated` belongs in `util` and it is `convert_to_stripe_object` that we should consider moving elsewhere. I will make a ticket.